VideoLocker/res/layout/fragment_discussion_search_posts.xml

```diff
@@ -17,5 +17,7 @@
         android:id="@+id/discussion_posts_listview"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        tools:listitem="@layout/row_discussion_thread" />
+        android:visibility="gone"
```

Comment:
You can also make this `invisible` instead of gone for symmetry with `fragment_discussion_thread_posts`, and because it's possible we might implement a footer here at some point too.